testsuite/compatibility-test/src/lib.rs

```diff
@@ -20,7 +17,7 @@ fn batch_update<'t>(
         ctx.swarm().upgrade_validator(*validator, version)?;
     }
 
-    let deadline = Instant::now() + Duration::from_secs(5);
+    let deadline = Instant::now() + Duration::from_secs(300);
```

Comment:
That's a very large deadline increase, any way we can keep it shorter?

Reply:
Maybe we could even have these are parameters to the test itself?

Reply:
yea, as you asked above, test failed at first and I suspect is because of deadline, so I increased this to a large number for better indicating, will make it to be a parameter for better adjustment

Reply:
Also, incase it wasn't clear what I meant, I was thinking about something like:

```rust
struct SimpleValidatorUpdate {
    batch_update_deadline: Duration,
}
```

And providing a constructor for setting that and a Default impl

Reply:
gotcha, will do